### DISCUSSION TITLE
[Feature] Quick sharing of a single photo/video

### DISCUSSION THREAD

**Discussion by dferrante** [immich-app/immich#14848] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This would be a sharing option within a single photo.  The quick share would:

- Only work on a singular image/video (not an album, etc)
- Create a copyable short URL similar to imgur, something like: https://<immich_url>/s/fKt5Sf.png or .mpg or whatever
- Photo/video would become public under this URL and stripped of EXIF/metadata

The use case for this is: I frequently want to share a single photo with others online in chat or elsewhere, and the regular share function has me share a link to a share album of one, which the receivers would have to click thru twice just to see one photo.  They would also see all the metadata, which I may not want to share.  Lately I've just been creating a share link, going to it myself and clicking thru, and finally sharing the raw URL to the image itself (which is long and ugly).  Doing this in one or two clicks would be so nice and useful.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
There's already a single-asset share option

![image](https://github.com/user-attachments/assets/b3bc3df2-330e-4b87-993f-a0987ec34385)

**Reply by dferrante** [reply]
that's what ive been using, yes, but going to the link created brings you to an album view: 
![image](https://github.com/user-attachments/assets/cc13fed6-4221-4b26-bc77-ced314b9bca9)

and then the person who gets the link has to click again to see the full size photo.  this also shares all the metadata of the image including GPS coords.  also the share album link does have oembed but sharing the link to one photo within that album does not.  this feature request is for a url direct to a raw image or video, stripped of EXIF/etc, similar to how imgur used to work.

**Reply by bo0tzz** [reply]
Then this is a dupe of #3556.

> shares all the metadata 

There's an option to turn this off in the share menu.
